cmd/par/main.go

```diff
@@ -387,7 +403,8 @@ func main() {
 		go func() {
 			<-c
 			pprof.StopCPUProfile()
-			os.Exit(1)
+			os.Exit(eMemoryError)
```

Comment:
I was wondering what this actually does :) Do you mean to not do this CPU profiling at all in 'lib mode'?

Reply:
The CPU profiling itself might make sense even in lib mode, but this code is implementing a signal handler so that hitting Ctrl-C writes out the profile before stopping the program. A library probably shouldn't install signal handlers, I think.